diff -uNr libftdi1-1.5/CMakeLists.txt libftdi1-1.5_patched/CMakeLists.txt
--- libftdi1-1.5/CMakeLists.txt	2020-07-07 21:32:55.000000000 +0200
+++ libftdi1-1.5_patched/CMakeLists.txt	2022-04-20 15:34:15.765574885 +0200
@@ -25,7 +25,7 @@
 endif(${CMAKE_BUILD_TYPE} STREQUAL Debug)
 
 # find libusb
-find_package ( USB1 REQUIRED )
+#find_package ( USB1 REQUIRED )
 include_directories ( ${LIBUSB_INCLUDE_DIR} )
 
 # Find Boost
