--- libftdi1-1.3/CMakeLists.txt	2017-08-07 14:47:09.742649368 +0200
+++ libftdi1-1.3_patched/CMakeLists.txt	2017-08-07 15:11:58.821437061 +0200
@@ -23,8 +23,8 @@
 endif(${CMAKE_BUILD_TYPE} STREQUAL Debug)
 
 # find libusb
-find_package ( USB1 REQUIRED )
-include_directories ( ${LIBUSB_INCLUDE_DIR} )
+find_package ( libusb CONFIG REQUIRED )
+include_directories ( ${libusb_INCLUDE_DIR} )
 
 # Find Boost (optional package)
 find_package(Boost)
@@ -187,14 +187,11 @@
 set ( LIBFTDI_INCLUDE_DIRS ${LIBFTDI_INCLUDE_DIR} )
 set ( LIBFTDI_LIBRARY ftdi1 )
 set ( LIBFTDI_LIBRARIES ${LIBFTDI_LIBRARY} )
-list ( APPEND LIBFTDI_LIBRARIES ${LIBUSB_LIBRARIES} )
 set ( LIBFTDI_STATIC_LIBRARY ftdi1.a )
 set ( LIBFTDI_STATIC_LIBRARIES ${LIBFTDI_STATIC_LIBRARY} )
-list ( APPEND LIBFTDI_STATIC_LIBRARIES ${LIBUSB_LIBRARIES} )
 if (FTDI_BUILD_CPP)
   set ( LIBFTDIPP_LIBRARY ftdipp1 )
   set ( LIBFTDIPP_LIBRARIES ${LIBFTDIPP_LIBRARY} )
-  list ( APPEND LIBFTDIPP_LIBRARIES ${LIBUSB_LIBRARIES} )
 endif ()
 set ( LIBFTDI_LIBRARY_DIRS ${libdir} )
 set ( LIBFTDI_ROOT_DIR ${prefix} )
diff -uNr libftdi1-1.3/examples/CMakeLists.txt libftdi1-1.3_patched/examples/CMakeLists.txt
--- libftdi1-1.3/examples/CMakeLists.txt	2017-08-07 15:10:33.891408113 +0200
+++ libftdi1-1.3_patched/examples/CMakeLists.txt	2017-08-07 15:11:58.821437061 +0200
@@ -21,16 +21,16 @@
     add_executable(eeprom eeprom.c)
 
     # Linkage
-    target_link_libraries(simple ftdi1)
-    target_link_libraries(bitbang ftdi1)
-    target_link_libraries(bitbang2 ftdi1)
-    target_link_libraries(bitbang_cbus ftdi1)
-    target_link_libraries(bitbang_ft2232 ftdi1)
-    target_link_libraries(find_all ftdi1)
-    target_link_libraries(serial_test ftdi1)
-    target_link_libraries(baud_test ftdi1)
-    target_link_libraries(stream_test ftdi1)
-    target_link_libraries(eeprom ftdi1)
+    target_link_libraries(simple ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(bitbang ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(bitbang2 ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(bitbang_cbus ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(bitbang_ft2232 ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(find_all ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(serial_test ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(baud_test ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(stream_test ftdi1 libusb::TARGET_libusb)
+    target_link_libraries(eeprom ftdi1 libusb::TARGET_libusb)
 
     # libftdi++ examples
     if(FTDI_BUILD_CPP)
diff -uNr libftdi1-1.3/ftdipp/CMakeLists.txt libftdi1-1.3_patched/ftdipp/CMakeLists.txt
--- libftdi1-1.3/ftdipp/CMakeLists.txt	2016-05-20 08:53:12.000000000 +0200
+++ libftdi1-1.3_patched/ftdipp/CMakeLists.txt	2017-08-07 15:15:40.038709949 +0200
@@ -33,7 +33,7 @@
     set_target_properties(ftdipp1 PROPERTIES CLEAN_DIRECT_OUTPUT 1)
 
     # Dependencies
-    target_link_libraries(ftdipp1 ftdi1 ${LIBUSB_LIBRARIES} ${BOOST_LIBRARIES})
+    target_link_libraries(ftdipp1 ftdi1 libusb::TARGET_libusb ${BOOST_LIBRARIES})
 
     
     install ( TARGETS ftdipp1
diff -uNr libftdi1-1.3/src/CMakeLists.txt libftdi1-1.3_patched/src/CMakeLists.txt
--- libftdi1-1.3/src/CMakeLists.txt	2017-08-07 15:11:58.825437157 +0200
+++ libftdi1-1.3_patched/src/CMakeLists.txt	2017-08-07 15:15:30.646486991 +0200
@@ -30,7 +30,7 @@
 
 
 # Dependencies
-target_link_libraries(ftdi1 ${LIBUSB_LIBRARIES})
+target_link_libraries(ftdi1 libusb::TARGET_libusb)
 
 install ( TARGETS ftdi1
           RUNTIME DESTINATION bin
@@ -40,7 +40,7 @@
 
 if ( STATICLIBS )
   add_library(ftdi1-static STATIC ${c_sources})
-  target_link_libraries(ftdi1-static ${LIBUSB_LIBRARIES})
+  target_link_libraries(ftdi1-static libusb::TARGET_libusb)
   set_target_properties(ftdi1-static PROPERTIES OUTPUT_NAME "ftdi1")
   set_target_properties(ftdi1-static PROPERTIES CLEAN_DIRECT_OUTPUT 1)
   install ( TARGETS ftdi1-static
